cmd/devdata/main.go

```diff
@@ -139,6 +139,7 @@ func main() {
 		})
 	}
 	if err := g.Wait(); err != nil {
+		fmt.Println(err)
```

Comment:
this is devdata so it's ok, but was this meant to be here

Reply:
Yeah... the panic wasn't really giving too much so I was hoping just printing the error would give the trace.